src/guides/v2.4/graphql/queries/products.md

```diff
@@ -986,6 +992,441 @@ By default, you cannot filter on the `color` attribute. [Filtering with custom a
 
 {% endcollapsible %}
 
+### Query with layered navigation and includeDirectChildrenOnly input filter applied
+
+The following query returns aggregations that filters on items with these characteristics:
+
+-  Women's Bottoms (category ID 22)
+-  In the price range of $40 - $49.99
+-  Comes in black (color 49)
+
+Because the includeDirectChildrenOnly input filter is set to true, the category aggregation in the response will include only the Women's Pants and Shorts categories, which are direct children of the Women's Bottoms category.
```

Comment:
```suggestion
Because the `includeDirectChildrenOnly` input filter is set to true, the category aggregation in the response will include only the Women's Pants and Shorts categories, which are direct children of the Women's Bottoms category.
```